js/pages/concept-sets/concept-manager.js

```diff
@@ -12,14 +12,14 @@ define([
 ], function (
 	ko,
 	view,
-	Component,
+	Page,
 	AutoBind,
 	vocabularyProvider,
 	commonUtils,
 	sharedState,
 	httpService,
 ) {
-	class ConceptManager extends AutoBind(Component) {
+	class ConceptManager extends AutoBind(Page) {
```

Comment:
Why haven't you created `async onPageCreated() {` and moved initial loading there then?